packages/job-worker/src/ingest/ingestRundownJobs.ts

```diff
@@ -84,13 +79,19 @@ export async function handleUserRemoveRundown(context: JobContext, data: UserRem
 				}
 			}
 		})
+	} else {
+		// Its a real rundown, so defer to the proper route for deletion
```

Comment:
To avoide future issues, I think it would be better to list the other alternatives in an `else if()`, ending with an `else assertNever(tmpRundown.source.type)`, this way one would be alerted of this code if adding other alternatives to `source.type` in the future.


Also (minor note): I think this comment is slightly confusing, what does "real" mean? (a testing-rundown is "real", isn't it?). Just changing this to "These follow the normal deletion flow" is good enough for me. :)

Reply:
The 'real' here means not sourced from a snapshot, which in other words means it was created by an ingest operation and not the snapshot restore operation. A testing rundown qualifies as 'real' here, as it was generation by Sofie from an IngestRundown, using the exact same logic to do so as an nrcs rundown.

This special case exists because of the comment at the top of this method:
```
	 * As the user requested a delete, it may not be from an ingest gateway, and have a bad relationship between _id and externalId.
	 * Because of this, we must do some more manual steps to ensure it is done correctly, and with as close to correct locking as is reasonable
```

But if you would prefer, that if could probably be changed to `tmpRundown._id !== getRundownId(context.studioId, tmpRundown.externalId)`, as that is really what this is guarding against.

Reply:
>But if you would prefer, that if could probably be changed to tmpRundown._id !== getRundownId(context.studioId, tmpRundown.externalId), as that is really what this is guarding against.

Yeah, that would make it easier to understand from an initial read, I think.